src/app/actions/actionTypes.js

```diff
@@ -0,0 +1,16 @@
+// Interactives
```

Comment:
Hi, I am wondering why you didn't use the current folders structure in the repository? the redux is stored in src/app/config. can you create new directory there for your object please?